.npmrc

```diff
@@ -0,0 +1 @@
+registry=https://registry.yarnpkg.com
```

Comment:
What's the purpose of this?

Reply:
Make sure that we use yarn registry when adding/upgrading packages.
Connecting to yarn registry or npm registry is incredibly slow in China, so a lot of developer in China ( including me ) changes their default registry to mirrors in China, like http://npm.taobao.org. Without `.npmrc`, these developers may accidentally use these Chinese mirrors and updates `yarn.lock` , like https://github.com/nuxt/nuxt.js/commit/7d8392f03683044da26b5a5865e5239491c07c53

Reply:
Make sure that we use yarn registry when adding/upgrading packages.
Connecting to yarn registry or npm registry is incredibly slow in China, so a lot of developer in China ( including me ) changes their default registry to mirrors in China, like http://npm.taobao.org. Without `.npmrc`, these developers may accidentally use these Chinese mirrors and updates `yarn.lock` , like https://github.com/nuxt/nuxt.js/commit/7d8392f03683044da26b5a5865e5239491c07c53

Reply:
Make sure that we use yarn registry when adding/upgrading packages.
Connecting to yarn registry or npm registry is incredibly slow in China, so a lot of developer in China ( including me ) changes their default registry to mirrors in China, like http://npm.taobao.org. Without `.npmrc`, these developers may accidentally use these Chinese mirrors and updates `yarn.lock` , like https://github.com/nuxt/nuxt.js/commit/7d8392f03683044da26b5a5865e5239491c07c53

Reply:
Thank you for the thorough explanation!